src/planscape/config/conditions.json

```diff
@@ -452,7 +474,8 @@
                   "data_year": "2021",
                   "reference_link": "https://data.fs.usda.gov/geodata/rastergateway/regional-resource-kit/docs/RRK_Metric_Dictionary_v2.pdf#page=21",
                   "min_value": 0,
-                  "max_value": 1
+                  "max_value": 1,
```

Comment:
max_value and range don't match up. also let's find a better unit

Reply:
same as probability max from above. i'm reserving all unclear unit asks for Pat and Nick